src/Raven.Studio/wwwroot/Content/css/_bs5.scss

```diff
@@ -45,6 +45,16 @@
     @import "../../../node_modules/boostrap5/scss/carousel";
     @import "../../../node_modules/boostrap5/scss/spinners";
     @import "../../../node_modules/boostrap5/scss/offcanvas";
+    //Custom components and mods
+    @import "../../../typescript/components/common/Background.scss";
+    @import "../../../typescript/components/common/Buttons.scss";
+    @import "../../../typescript/components/common/Checkbox.scss";
+    @import "../../../typescript/components/common/HrHeader.scss";
+    @import "../../../typescript/components/common/Icon.scss";
+    @import "../../../typescript/components/common/LazyLoad.scss";
+    @import "../../../typescript/components/common/NodeSet.scss";
+    @import "../../../typescript/components/common/StickyHeader.scss";
```

Comment:
it should be imported from sticky header, not from this global style. 

Same for other items.